test/cli/autocorrect-kwsplat/test.out

```diff
@@ -119,6 +119,8 @@ test.rb:29: Expected `String` for keyword parameter `x` but found `T.nilable(Int
     test.rb:29:
     29 |optional_x(0, 1, 2, **nilble_opts)
                             ^^^^^^^^^^^^^
+  Detailed explanation:
+    `Integer` (the left side of the `T.any`) is not a subtype of `String`
```

Comment:
I think this reads a little odd in the `T.nilable` case? 

"The left side of the T.any" – what T.any? The user didn't write T.any here.